src/IECoreVDB/VDBObject.cpp

```diff
@@ -195,17 +201,17 @@ void VDBObject::render( IECoreScene::Renderer *renderer ) const
 
 IECore::CompoundObjectPtr VDBObject::metadata( const std::string &name )
 {
-	openvdb::GridBase::Ptr grid = findGrid( name );
+	CompoundObjectPtr metadata = new CompoundObject();
+
+	openvdb::GridBase::Ptr grid = findGrid ( name );
```

Comment:
Looks like this is loading the whole grid, when just the metadata would be sufficient? We could leave this as later work, as I think the problem predates this PR.